pkg/resource/parameter_group/hooks.go

```diff
@@ -15,6 +15,7 @@ package parameter_group
 
 import (
 	"context"
+	ackutil "github.com/aws-controllers-k8s/runtime/pkg/util"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Yeah it was automatically added there. I will change the order manually.

Reply:
Actually I will fix it in the next PR. Looks like most hooks have import order issue.